Packs/XMCO/Integrations/Serenety/Serenety.yml

```diff
@@ -0,0 +1,61 @@
+fromversion: 6.10.0
+category: Vulnerability Management
+commonfields:
+  id: Serenety
+  version: -1
+configuration:
+- display: XMCO LePortail instance URL
+  defaultvalue: https://leportail.xmco.fr/
+  name: url
+  required: true
+  type: 0
+- displaypassword: API Key
+  name: api_key
+  required: true
+  type: 9
+  hiddenusername: true
+  section: Connect
```

Comment:
For this key to have an effect, the section order needs to be defined. You can find out more about configuration sections [here](https://xsoar.pan.dev/docs/integrations/yaml-file#configurations-sections).